sales.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Salmon Cookies</title>
+</head>
+<body>
+  <header>
+    <h1>Salmon Cookies</h1>
+    <img src='salmon.png'/>
+  </header>
+  <nav><a href='index.html'>Home</a>
+  </nav>
+  <section>
+    <table>
+ <tr id= 'sales-estimates'></tr>
```

Comment:
Try adding a thead, tbody, and tfoot to the table.  These are built-in tags that will allow you to append a header table row to the thead, a number of rows containing the data you are dispaying to the tbody, and tfoot will allow you to append any additional information such as a totals row .